src/content/third_party_migration/vendors/redhat-json.json

```diff
@@ -0,0 +1,30 @@
+{
+    "vendor": "redhat-json",
+    "description": "RedHat JSON",
+    "remotes": [
+        {
+            "url": "https://access.redhat.com/labs/securitydataapi/cve.json?after={after}&per_page={per_page}&page={page}",
+            "request": "api",
+            "type": "json",
+            "parameters": {
+                "after": {
+                    "type": "fixed",
+                    "description": "Date in ISO 8601 format (YYYY-MM-DD or YYYYMMDD)",
+                    "value": "1999-01-01"
```

Comment:
- To make the schema simpler change `value` from string to array:
    - `"value":["1999-01-01"]`
- Change the type name from `fixed` to `list` as is more descriptive of what it is.